packages/botonic-plugin-ner/src/utils/environment-utils.ts

```diff
@@ -0,0 +1,24 @@
+import { Locale } from '@botonic/nlp/dist/types'
+import { loadLayersModel } from '@tensorflow/tfjs'
+import axios from 'axios'
+
+import {
+  ASSETS_DIR,
+  CONFIG_FILENAME,
+  MODEL_FILENAME,
+  MODELS_DIR,
+  NER_DIR,
+} from '../constants'
+import { ModelInfo } from '../types'
+
+export async function getModelInfo(locale: Locale): Promise<ModelInfo> {
+  const isProd = process.env.STATIC_URL !== undefined
```

Comment:
Env vars are shared for all applications on a system. Use a name as distinct as possible to give it as much semantincs as possible and to avoid collisions (eg. use BOTONIC_NLP_MODEL_URL instead of STATIC_URL)

Reply:
We would have to modify backend to change this env variable name. @marcrabat